### PR TITLE
Added a check in the create Field plot method on the face ids. 

### PR DESCRIPTION
**Pull request by maxcapodi78**
If the face id belongs to a non model object the syntax has to be changed

### CODE REVIEW

File: pyaedt/modules/solutions.py [hunk 1]
```diff
@@ -2285,11 +2285,29 @@ def plotGeomInfo(self):
...
+                except:
+                    pass
+            info.append("Surface")
+            if model_faces:
+                info.append("FacesList")
+                info.append(len(model_faces))
+                for index in model_faces:
+                    info.append(index)
+            if nonmodel_faces:
+                info.append("NonModelFaceList")
```
Comment (MaxJPRey) [inline pyaedt/modules/solutions.py:2307]:
```suggestion
                info.append("NonModelFacesList")
```

Comment (maxcapodi78) [reply pyaedt/modules/solutions.py:2307]: @MaxJPRey that is AEDT API syntax. We cannot choose

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

File: pyaedt/modules/solutions.py (hunk 1 above)
Comment (MaxJPRey) [reply pyaedt/modules/solutions.py:2307]: OK. Got it. Thanks.

**Pull request merged**